--- a/data/carla.desktop	2017-05-10 21:44:17.000000000 +0200
+++ b/data/carla.desktop	2017-05-12 20:18:17.619781928 +0200
@@ -8,7 +8,7 @@
 Icon=carla
 Terminal=false
 Type=Application
-Categories=AudioVideo;AudioEditing;Qt;
+Categories=AudioVideo;AudioVideoEditing;Qt;
 MimeType=application/x-carla-project;
 Version=1.0
 X-NSM-capable=true

--- a/data/carla-control.desktop	2017-05-12 20:18:59.044426579 +0200
+++ b/data/carla-control.desktop	2017-05-12 20:18:48.747514830 +0200
@@ -8,5 +8,5 @@
 Icon=carla-control
 Terminal=false
 Type=Application
-Categories=AudioVideo;AudioEditing;Qt;
+Categories=AudioVideo;AudioVideoEditing;Qt;
 Version=1.0
